apps/site/src/lib/components/Navbar/Navbar.tsx

```diff
@@ -76,10 +75,18 @@ function Navbar({ identity }: NavbarProps) {
 					<NavLinkItem href="/">Schedule</NavLinkItem>
 					<NavLinkItem href="/">Resources</NavLinkItem>
 					<NavLinkItem href="/">Stage</NavLinkItem> */}
-					{!status && <NavLinkItem href="/apply">Apply</NavLinkItem>}
-					{status !== null && (
-						<NavLinkItem href="/portal">Portal</NavLinkItem>
-					)}
+					{!status &&
+						(deadlinePassed ? (
+							<p
+								className={clsx(styles.navMenuLink, styles.disabledLink, "m-0")}
+							>
+								Applications have closed
+							</p>
+						) : (
+							<NavLinkItem href="/apply">Apply</NavLinkItem>
```

Comment:
Issue: I think we can just not show the `Apply` link when the deadline has passed, mainly because now, this element isn't a link (and therefore shouldn't be in the navbar).